supplementary/hardware-enablement.rst

```diff
@@ -45,7 +45,7 @@ For all ARM based smart phones and tablets this should be inside
 
    root/arch/arm/boot/dts/
 
-or if your using 64-bit arm64 (e.g. BQ Aquaris M10) all files are sorted by manufacturer inside
+or if your device is using a 64-bit (arm64) processor (e.g. BQ Aquaris M10) all files are sorted by manufacturer inside
```

Comment:
I would say "is running on", the wording is quite unusual for me

Reply:
I think the original author wanted to say "if you're using a 64-bit processor", which is why it got so weird now

Reply:
Nvm, an English-man said that it sounds fine

Reply:
"or if your device is uses an arm64 processor (e.g. BQ Aquaris M10) all the files are sorted by the manufacturer in :"
this is what i think it should be